test/integration/user-management-api/providers_test.rb

```diff
@@ -17,7 +17,8 @@ def setup
 
     account = JSON.parse(response.body)['account']
 
-    assert_equal @provider.admin_domain, account['admin_domain']
+    assert_equal @provider.internal_admin_domain, account['admin_domain']
+    assert_equal @provider.external_admin_domain, account['admin_domain']
```

Comment:
why internal and external domains are the same?

Reply:
So you used `internal/external` based on your understanding of semantics at the particular place?